### ISSUE TITLE
Missing start activity after conversion to DataFrame and CSV export/import

### ISSUE THREAD

**Issue by goto-loop** [process-intelligence-solutions/pm4py#340]
I'm currently working with the [Sepsis Cases dataset](https://data.4tu.nl/articles/dataset/Sepsis_Cases_-_Event_Log/12707639), which comes originally in XES format. I've noticed that the data changes when converting the EventLog object to a DataFrame, writing it to a CSV file and then reading it again.

The issue can be reproduced with the following code (the dataset can be downloaded at the link provided above):
```python
import pandas as pd
import pm4py

log = pm4py.read_xes("Sepsis Cases - Event Log.xes.gz")
print(pm4py.get_start_activities(log))

log_df = pm4py.convert_to_dataframe(log)
print(pm4py.get_start_activities(log_df))

log_df.to_csv("Sepsis Cases - Event Log.csv", index=False)
log_df_import = pd.read_csv("Sepsis Cases - Event Log.csv")
print(pm4py.get_start_activities(log_df_import))
```

I get an output like this:
```python
{'ER Registration': 995, 'IV Liquid': 14, 'ER Triage': 6, 'CRP': 10, 'ER Sepsis Triage': 7, 'Leucocytes': 18}
{'ER Registration': 995, 'Leucocytes': 18, 'IV Liquid': 14, 'CRP': 10, 'ER Sepsis Triage': 7, 'ER Triage': 6}
{'ER Registration': 994, 'Leucocytes': 18, 'IV Liquid': 14, 'CRP': 10, 'ER Sepsis Triage': 7, 'ER Triage': 6}
```

Notice the change in the `ER Registration` value on the third line. I would expect the output to be exactly the same in all three cases.

EDIT: Using Python 3.10.5, pandas 1.4.3 and pm4py 2.2.23.1.

**Comment by fit-alessandro-berti** [comment]
Dear goto-loop,

I would like to invite you to use the function pm4py.format_dataframe which is essential in the current version to format the timestamp column and sort the events properly. If this does not solve your issue, it might be a timestamp issue (several events in the case have the same timestamp)

**Comment by goto-loop** [comment]
Hi Alessandro,

thanks for the suggestion!

However, I've been able to track the issue down and it's not related to pm4py. I've noticed there is exactly one case missing in the imported DataFrame and it's missing completely. The missing case has the name "NA". When I replace "NA" with "ABCD" before doing the export/import, all events are there. As it turns out, pandas' `read_csv()` drops all rows containing values that are recognized as NA/NaN so when I pass `na_filter=False`, all events are there.
